backend/clickhouse/log_row.go

```diff
@@ -0,0 +1,104 @@
+package clickhouse
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/highlight/highlight/sdk/highlight-go"
+	"github.com/influxdata/influxdb-client-go/v2/log"
+)
+
+type LogRowPrimaryAttrs struct {
+	Timestamp       time.Time
+	ProjectId       uint32
+	TraceId         string
+	SpanId          string
+	SecureSessionId string
+}
+
+type LogRow struct {
+	LogRowPrimaryAttrs
+	UUID           string
+	TraceFlags     uint32
+	SeverityText   string
+	SeverityNumber int32
+	ServiceName    string
+	Body           string
+	LogAttributes  map[string]string
+}
+
+func NewLogRow(attrs LogRowPrimaryAttrs, opts ...LogRowOption) *LogRow {
+	logRow := &LogRow{
+		LogRowPrimaryAttrs: LogRowPrimaryAttrs{
+			Timestamp:       attrs.Timestamp,
+			TraceId:         attrs.TraceId,
+			SpanId:          attrs.SpanId,
+			ProjectId:       attrs.ProjectId,
+			SecureSessionId: attrs.SecureSessionId,
+		},
+		UUID:           uuid.New().String(),
+		SeverityText:   "INFO",
+		SeverityNumber: int32(log.InfoLevel),
```

Comment:
this imported `github.com/influxdata/influxdb-client-go/v2/log` - is that consistent with the otel value for the info log level?

Reply:
Good catch. The auto-import caught the wrong import.

Reply:
fe8345b0b05acb8bc69a471bb9ede7dc0d054dfa